cloud/deployment/deployment.go

```diff
@@ -424,6 +401,7 @@ func Update(deploymentID, label, ws, description string, schedulerAU, schedulerR
 
 	// update deployment
 	d, err := client.UpdateDeployment(deploymentUpdate)
+	fmt.Println(d.ID)
```

Comment:
Do we need this, or was this for debugging?

Reply:
just debugging I am removing